src/tools/kg_tools.py

```
@@ -3,12 +3,17 @@
 """
 import os
 import json
+import logging
 from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)